assets/css/bootstrap/bootstrap.css

```diff
@@ -400,6 +400,18 @@ h6, .h6 {
   font-size: 1rem;
 }
 
+span.wpcf7-list-item {
+  display: block;
+  margin-left: 0;
```

Comment:
Better to use the bootstrap classes for this.

Reply:
The wpcf7-list-item spans are generated by the Contact Forms 7 plugin, and there's no way to assign them a class of ml-0 (unless we edit the CF7 plugin code). I tried assigning the ml-0 class in the contact form designer (using [checkbox class:ml-0]), but it doesn't have the intended effect.